test-lib.sh

```diff
@@ -1186,4 +1186,58 @@ function ct_timestamp_diff() {
   date -u -d "0 $final_date seconds - $start_date seconds" +"%H:%M:%S"
 }
 
+# ct_get_certificate_timestamp
+# ----------------------------
+# Looks into a running container into a specified file (certificate) and extracts
+# a notBefore date.
+# Argument: container - ID of a running container
+# Argument: path - path inside the running container
+# Returns: timestamp (seconds since Unix era) for the certificate generation
+function ct_get_certificate_timestamp() {
+  local container=$1
+  local path=$2
+set -x
+which openssl >&2
```

Comment:
Is this formatting intentional?

Reply:
This looks like a not-finished PR, will take a look and fix the formatting.